insights/combiners/tests/test_httpd_conf_tree.py

```diff
@@ -91,6 +91,21 @@
 </IfModule>
 '''.strip()
 
+HTTPD_CONF_WITH_NONSTANDARD_INCLUDE_PATH = '''
```

Comment:
Oh, this is from another branch in my testing, but this is not a failing test, it actually does what I expected.